index.html

```diff
@@ -291,6 +292,109 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer class="footer">
+      <section id="contact" class="container contact">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          action="https://formspree.io/f/mvodnloy"
+          method="post"
+          class="form"
+        >
+          <input
+            name="fullname"
+            type="text"
+            placeholder="John Doe"
+            maxlength="30"
+            aria-required="true"
+            required
+          />
+          <input
+            name="email"
+            type="email"
+            placeholder="Email Address"
+            aria-required="true"
+            required
+            pattern="^[a-z0-9]+@(.+)$"
+          />
+          <textarea
+            name="message"
+            placeholder="Hello Mariane. Glad to see you "
+            maxlength="500"
+            aria-required="true"
+            required
+          ></textarea>
+          <button class="btn btn-form" type="submit">Get In Touch</button>
+
+          <span class="error-msg"></span>
+        </form>
+      </section>
+
+      <div class="social">
```

Comment:
- [ ] Required: Kindly add a horizontal line above the social links as you have noticed as per the contact form for mobile version in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
<img width="267" alt="Screenshot 2022-03-10 at 12 29 29 AM" src="https://user-images.githubusercontent.com/70269507/157539329-c2430e0d-821f-4f17-a5c4-0af5bd8562e1.png">